core/domain/user_services.py

```diff
@@ -1891,22 +1893,23 @@ def get_human_readable_user_ids(user_ids, strict=True):
     Returns:
         list(str). List of usernames corresponding to given user_ids. If
         username does not exist, the corresponding entry in the returned
-        list is the user's truncated email address.
+        list is the user's truncated email address. If the user is scheduled to
+        be deleted USER_IDENTIFICATION_FOR_USER_BEING_DELETED is returned.
```

Comment:
Perhaps change "USER_IDENTIFICATION" to "LABEL" to avoid confusion with the concept of user_id.